src/Makefile

```
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
-obj-m := dattobd.o
+obj-m += dattobd.o
+dattobd-objs := sset_list.o bio_helper.o bio_list.o sset_queue.o tracing_params.o bio_queue.o mrf.o tracer_helper.o userspace_copy_helpers.o task_helper.o filesystem.o cow_manager.o snap_handle.o blkdev.o ioctl_handlers.o tracer.o system_call_hooking.o module_control.o module_threads.o proc_seq_file.o snap_ops.o paging_helper.o
```

does something like
```suggestion
dattobd-objs := $(shell find . -maxdepth 1 -name '*.c' | sed s/\.c/\.o/')
```
work instead?